src/block/button/save.js

```diff
@@ -25,25 +25,36 @@ import { useBlockProps } from '@wordpress/block-editor'
  * Internal dependencies
  */
 import { ButtonStyles } from './style'
+import { applyFilters } from '@wordpress/hooks'
 
 export const Save = props => {
 	const {
+		attributes,
```

Comment:
You extracted into its own variable `attributes`, so now it will not be anymore part of the `propsToPass` during the deconstruction. This will lead to unexpected issues